src/debug/createdump/CMakeLists.txt

```diff
@@ -20,7 +20,7 @@ include_directories(BEFORE ${VM_DIR})
 
 add_definitions(-DPAL_STDCPP_COMPAT=1)
 
-add_compile_options(-fPIE)
+add_compile_options(-fPIC)
```

Comment:
The `createdump` is an executable, so  it should be compiled with `-fPIE`. I don't see where we link in code compiled with `-fPIC`. The `createdump_lib` is linked with '-fPIE' and the `mscordaccore` is a shared library.
Looking at the command line of compiling one of the sources above, I also don't see `-fPIC` being specified there, so it is not clear to me why does GCC complain.

Reply:
I agree, I'll back out this change to figure out what exactly is happening.